packages/wdio-config/src/utils.ts

```diff
@@ -220,10 +221,10 @@ export function validateConfig<T>(defaults: Options.Definition<T>, options: T, k
     return params
 }
 
-export function loadTypeScriptCompiler () {
+export function loadTypeScriptCompiler (tsNodeOpts?: WebdriverIO.TsNodeOpts) {
```

Comment:
```suggestion
export function loadTypeScriptCompiler (tsNodeOpts?: RegisterOptions) {
```

Reply:
Great call, was also able to get rid of that "as any" 💃